lib/utils.js

```diff
@@ -0,0 +1,37 @@
+const { curry } = require('ramda');
+const { KEY_START, KEY_END, RSA_KEY_START, RSA_KEY_END } = require('./constants');
+const domainRegTest = /^[a-z0-9-]+$/;
+
+const tryParseJson = (keyjson) => {
+  if (typeof keyjson === 'string') {
+    try {
+      keyjson = JSON.parse(keyjson);
+    } catch (e) {
+      throw Error(`Could not parse json properly: ${e.message}`);
+    }
+  }
+  return keyjson;
+};
+
+const isSslKey = (key) => {
+  return !(key.startsWith(KEY_START) && key.endsWith(KEY_END)) && !(key.startsWith(RSA_KEY_START) && key.endsWith(RSA_KEY_END))
+};
+
+const isDomainMatch = curry((toCheck, base) => {
+  if (base === toCheck) return true;
```

Comment:
@spacetc62 surprised you don't have anything to say about the missing brackets here. Or maybe that's what drove the linter suggestion 😉.

Reply:
I figured if I'm writing this from scratch I can use my one line if statements without brackets. Just like I like.